### ISSUE TITLE
Error reported in IDE but not in CLI

### ISSUE THREAD

**Issue by abhinav-enboarder** [levibuzolic/eslint-plugin-no-only-tests#50]
I have been using this package in a JavaScript repo but the similar config does not work in a TypeScript based repo.

I am getting errors reported in VS Code but when running the eslint CLI, no errors are reported.

Version used `"eslint-plugin-no-only-tests": "3.3.0"`
Here is my `.eslintrc.json`
```json
{
    "env": {
        "es6": true,
        "node": true,
        "jest": true
    },
    "extends": [
        "airbnb-base",
        "prettier",
        "plugin:@typescript-eslint/recommended",
        "plugin:@typescript-eslint/eslint-recommended",
        "plugin:import/typescript"
    ],
    "plugins": ["prettier", "no-only-tests", "@typescript-eslint"],
    "parser": "@typescript-eslint/parser",
    "parserOptions": {
        "ecmaVersion": 12
    },
    "settings": {
        "import/parsers": {
            "@typescript-eslint/parser": [".ts"]
        }
    },
    "rules": {
        "no-only-tests/no-only-tests": ["error"],

```
